### ISSUE TITLE
[False-Negative] Non-optimal regular expression, s PCRE_DOTALL

### ISSUE THREAD

**Issue by puggan** [kalessil/phpinspectionsea#1135]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | False-Negative |
| Plugin         | Php Inspections (EA Ultimate) v2.0.11 |
| Language level | PHP5.6 |

Current behaviour (description/screenshot:)
--
"Non-optimal regular expression" on 2nd regexp.
![screenshot from 2018-12-07 12-30-35](https://user-images.githubusercontent.com/866668/49645379-f538c700-fa1b-11e8-8a78-66491d6e4a68.png)
```
    $t = '';
    preg_match('/[a-z]./s', $t);
    preg_match('/[a-z]/s', $t);
    preg_match('/[a-z.]/s',$t );
```

Expected behaviour
--
"Non-optimal regular expression" on 2nd and 3th regexp.

Environment details
---
PhpStorm 2018.3
Build #PS-183.4284.150, built on November 21, 2018
JRE: 1.8.0_152-release-1343-b15 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.18.0-12-generic

**Comment by kalessil** [comment]
Ooops, thank you for reporting @puggan

**Comment by kalessil** [comment]
Fixed!
